cheatsheets/TLS_Cipher_String_Cheat_Sheet.md

```diff
@@ -58,24 +58,27 @@ This table lists the names used by IANA and by openssl in brackets `[]`. Additio
 <!-- markdownlint-disable MD033 -->
 | Cipher name:<br>IANA, [OpenSSL] | Cipher HEX value | Advanced<br>(A) | Broad Compatibility<br>(B) | Widest Compatibility<br>(C) | Legacy<br>(D) |
 | --- | :---: | :---: | :---: | :---: | :---: |
-| `TLS_DHE_RSA_WITH_AES_256_GCM_SHA384`,<br>[`DHE-RSA-AES256-GCM-SHA384`] | 0x009f | 1 | 1 | 1 | 1 |
-| `TLS_DHE_RSA_WITH_AES_128_GCM_SHA256`,<br>[`DHE-RSA-AES128-GCM-SHA256`] | 0x009e | 2 | 2 | 2 | 2 |
-| `TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384`,<br>[`ECDHE-RSA-AES256-GCM-SHA384`] | 0xc030 | 3 | 3 | 3 | 3 |
-| `TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256`,<br>[`ECDHE-RSA-AES128-GCM-SHA256`] | 0xc02f | 4 | 4 | 4 | 4 |
-| `TLS_DHE_RSA_WITH_AES_256_CBC_SHA256`,<br>[`DHE-RSA-AES256-SHA256`] | 0x006b |  | 5 | 5 | 5 |
-| `TLS_DHE_RSA_WITH_AES_128_CBC_SHA256`,<br>[`DHE-RSA-AES128-SHA256`] | 0x0067 |  | 6 | 6 | 6 |
-| `TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384`,<br>[`ECDHE-RSA-AES256-SHA384`] | 0xc028 |  | 7 | 7 | 7 |
-| `TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256`,<br>[`ECDHE-RSA-AES128-SHA256`] | 0xc027 |  | 8 | 8 | 8 |
-| `TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA`,<br>[`ECDHE-RSA-AES256-SHA`] | 0xc014 |  |  | 9 | 9 |
-| `TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA`,<br>[`ECDHE-RSA-AES128-SHA`] | 0xc013 |  |  | 10 | 10 |
-| `TLS_RSA_WITH_AES_256_GCM_SHA384`,<br>[`AES256-GCM-SHA384`] | 0x009d |  |  |  | 11 |
-| `TLS_RSA_WITH_AES_128_GCM_SHA256`,<br>[`AES128-GCM-SHA256`] | 0x009c |  |  |  | 12 |
-| `TLS_RSA_WITH_AES_256_CBC_SHA256`,<br>[`AES256-SHA256`] | 0x003d |  |  |  | 13 |
-| `TLS_RSA_WITH_AES_128_CBC_SHA256`,<br>[`AES128-SHA256`] | 0x003c |  |  |  | 14 |
-| `TLS_RSA_WITH_AES_256_CBC_SHA`,<br>[`AES256-SHA`] | 0x0035 |  |  |  | 15 |
-| `TLS_RSA_WITH_AES_128_CBC_SHA`,<br>[`AES128-SHA`] | 0x002f |  |  |  | 16 |
-| `TLS_DHE_RSA_WITH_AES_256_CBC_SHA`,<br>[`DHE-RSA-AES256-SHA`] | 0x0039 |  |  | 11 | 17 |
-| `TLS_DHE_RSA_WITH_AES_128_CBC_SHA`,<br>[`DHE-RSA-AES128-SHA`] | 0x0033 |  |  | 12 | 18 |
+| `TLS_AES_256_GCM_SHA384` *(TLSv1.3)*,<br>[`TLS_AES_256_GCM_SHA384`] | 0x1302 | 1 | 1 | 1 | 1 |
```

Comment:
I don't believe these priorities still stick as they are, especially after your update. Did you validate the priority vs the following [mozilla URL](https://wiki.mozilla.org/Security/Server_Side_TLS#Cipher_names_correspondence_table)?

Reply:
Do you mean the existing chipers or the TLSv1.3 ones?
The new ciphers are not present at Mozilla yet, however the existing ones require a refactor if the Mozilla list is the example to follow. I think the sort of priorities is a different issue.

Reply:
True, let's leave that for another issue. Thank you for the PR.